src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -3491,10 +3489,9 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
           vparams.lengthIs <= MaxFunctionArity &&
           (formal baseType FunctionClass(vparams.length) match {
             case TypeRef(_, _, formalargs) =>
-              ( exists2(formalargs, vparams)((formal, vparam) =>
-                        vparam.tpt.isEmpty && (tparams exists formal.contains))
-                && (tparams forall isLowerBounded)
-              )
+              exists2(formalargs, vparams) { (formal, vparam) =>
+                vparam.tpt.isEmpty && (tparams exists formal.contains)
```

Comment:
I understand we want to get rid of infix notation for methods. 

```suggestion
                vparam.tpt.isEmpty && tparams.exists(formal.contains)
```

Reply:
Yeah there are many more of those around but I can change the places I'm touching 👍